python/src/Dirac_doc.i.in

```diff
@@ -1,6 +1,8 @@
 %feature("docstring") OT::Dirac
 "Dirac distribution.
 
+This is intended to be used for modelling deterministic parameters.
```

Comment:
"A possible usage is the modelling of deterministic parameters"->the user is free to use it the way she want ;-)